.changeset/moody-goats-cry.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": minor
+---
+
+Adds `wrangler triggers deploy --force-subdomain-deploy` flag.
```

Comment:
nit: still think the following wording better express what's new (`force-subdomain-deploy`) and being added to the existing `wrangler triggers deploy` command.

```suggestion
Adds a `--force-subdomain-deploy` flag to the `wrangler triggers deploy` command.
```